sylt-std/src/sylt.rs

```diff
@@ -271,7 +257,10 @@ pub fn inf<'t>(values: &[Value], _ctx: RuntimeContext<'t>) -> Result<Value, Runt
         [x] => {
             let t: Type = Type::from(&*x);
             let x = x.clone();
-            Ok(Value::Iter(t, Rc::new(RefCell::new(Box::new(move || Some(x.clone()))))))
+            Ok(Value::Iter(
+                t,
+                Rc::new(RefCell::new(Box::new(move || Some(x.clone())))),
+            ))
```

Comment:
We can actually remove everything that has with Iter to do - I've been meaning to do that for some time.